oneBarangay/static/assets/gulpfile.js

```diff
@@ -16,92 +16,92 @@
 
 */
 
-const autoprefixer = require("gulp-autoprefixer");
-const browserSync = require("browser-sync").create();
-const cleanCss = require("gulp-clean-css");
-const del = require("del");
-const htmlmin = require("gulp-htmlmin");
-const cssbeautify = require("gulp-cssbeautify");
-const gulp = require("gulp");
-const npmDist = require("gulp-npm-dist");
-const sass = require("gulp-sass")(require("node-sass"));
-const wait = require("gulp-wait");
-const sourcemaps = require("gulp-sourcemaps");
-const fileinclude = require("gulp-file-include");
-const rename = require("gulp-rename");
+const autoprefixer = require('gulp-autoprefixer')
+const browserSync = require('browser-sync').create()
+const cleanCss = require('gulp-clean-css')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
const cleanCss = require("gulp-clean-css")
```